docs/content/gameobject/reading-and-writing-component-data.md

```diff
@@ -2,23 +2,23 @@
 
 -----
 
-## GameObject: Reading and writing SpatialOS component data
+## GameObject: Reading and writing SpatialOS component property data
 
 SpatialOS schema defines what objects in your game interact with SpatialOS. Find out about [what schema does](https://docs.improbable.io/reference/latest/shared/glossary#schema) and how to implement schema via the [schemalang reference](https://docs.improbable.io/reference/latest/shared/schema/reference) in the SpatialOS documentation.
 
-We provide code-generated Reader and Writer objects for interacting with SpatialOS component data. 
+We provide code-generated Reader and Writer interfaces for interacting with SpatialOS component [properties](https://docs.improbable.io/reference/latest/shared/glossary#property). 
 
-For every component defined in SpatialOS schema, the code generation creates a pair of Reader and Writer interfaces within:
+For every component defined in SpatialOS schema, we generate a pair of Reader and Writer interfaces within:
 
-* `Generated.<namespace of schema component>.<component name>.Requirables.Reader`
+* `Generated.<namespace of schema component>.<component name>.Requirable.Reader`
+* `Generated.<namespace of schema component>.<component name>.Requirable.Writer` 
 
-* `Generated.<namespace of schema component>.<component name>.Requirables.Writer` 
+You can access Reader and Writer interfaces by declaring a field in your MonoBehaviour and decorating it with the `[Require]` attribute. `[Require]` fields are automatically injected and removed based on certain requirements:
```

Comment:
I think highlighting that they are interfaces is confusing. If someone tells me they generated an interface my first question will be what implements it. Do I need to write something or is something provided and I just inject it? It would take me a moment to realise that accessing an interface just meant injecting something pre-made that implements it.

Reply:
I'd also really like to get rid of the interfaces and use concrete objects long term.

Reply:
Are you ok with me replacing interface with "API"?

Reply:
Discussed offline. Changed "Reader and Writer interface" to "Readers and Writers".